tests/test_bar_data.py

```diff
@@ -100,15 +101,12 @@ class TestMinuteBarData(WithBarDataChecks,
 
     @classmethod
     def make_equity_info(cls):
-        return pd.DataFrame.from_dict(
-            {
-                sid: {
-                    'start_date': cls.SIM_PARAMS_START,
-                    'end_date': cls.SIM_PARAMS_END,
-                    'symbol': "ASSET{0}".format(sid)
-                } for sid in [1, 2, 3, 4, 5]
-            },
-            orient='index',
+        sids = [1, 2, 3, 4, 5]
+        return make_simple_equity_info(
+            sids,
+            cls.SIM_PARAMS_START,
+            cls.SIM_PARAMS_END,
+            map('ASSET{}'.format, sids),
```

Comment:
This is going to to generate a DataFrame full of map objects in Python 3.  I have made this mistake several times.